packages/image_picker/image_picker_macos/example/macos/Runner/AppDelegate.swift

```diff
@@ -5,9 +5,13 @@
 import Cocoa
 import FlutterMacOS
 
-@NSApplicationMain
+@main
 class AppDelegate: FlutterAppDelegate {
   override func applicationShouldTerminateAfterLastWindowClosed(_ sender: NSApplication) -> Bool {
     return true
   }
+
+  override func applicationSupportsSecureRestorableState(_ app: NSApplication) -> Bool {
+    return true
+  }
```

Comment:
This has been automated somehow, should I revert?

Reply:
This is due to running `flutter run -d macos` when using the `main` channel (automated):

```console
macos/Runner/AppDelegate.swift does not override
applicationSupportsSecureRestorableState. Updating.
```